index.js

```diff
@@ -80,11 +80,24 @@ function encodeToDialect(dialect, content, model) {
  * @method getSequelizeTypeFromJoi
  * @param  {String}          dialect Underlying system that we're writing to
  * @param  {String}          type    Joi type
+ * @param  {Array}           rules   Joi rules array
  * @return {SequelizeType}           Type to use in Sequelize
  */
-function getSequelizeTypeFromJoi(dialect, type) {
+function getSequelizeTypeFromJoi(dialect, type, rules) {
+    let length;
+
+    // Get the column length if length/max rule is included in dataschema
+    if (rules) {
```

Comment:
Yes. But `joi` doesn't create the `rules` key when schema has no rule such as length.
```
> require('joi').string().describe()
{ type: 'string', invalids: [ '' ] }
```
I'll change L167 not to pass undefined to `getSequelizeTypeFromJoi`.